setup.py

```diff
@@ -19,7 +19,8 @@
                             'pubs.plugs', 'pubs.plugs.alias'],
     scripts      = ['pubs/pubs'],
 
-    install_requires = ['pyyaml', 'bibtexparser', 'python-dateutil', 'requests', 'configobj',
+    install_requires = ['pyyaml', 'bibtexparser', 'python-dateutil', 'requests',
+                        'configobj', 'argcomplete',
```

Comment:
(This commit should not be part of this PR.)
Anyway, since the auto-completion still requires manual configuration I don't believe it worth it to include it as a requirement. We should however definitely add it as an `extras_require`.